services/frontend-v3/src/utils/login.js

```diff
@@ -43,6 +43,7 @@ const profileExist = async (userInfo, axios) => {
       email,
       status,
       signupStep,
+      isPrivacyAccepted: false,
```

Comment:
Should remove this, since it is not doing anything now in the slice

Reply:
removed